### PR TITLE
(PUP-7983) Enable testing system openssl linked PA on RHEL7 in CI pipes

### PR DESCRIPTION
**Pull request by jsane**
Update openssl package to desired version on RHEL7 hosts since the default
version installed in images used by CI/VMPooler are not compatible with
agent setup and operation causing failures. Note that PA is built on centos7
which is also used to test on rhel platforms.

Related to PA-1580 (part of epic PA-1608). Has been tested in PA CI - logs would be attached to PR for PA-1264 (enables linking PA against system openssl).
This does not impact regular agent build using vendored openssl which disregard system
openssl.

### CODE REVIEW

**Comment by jsane** [comment]
I also pushed changes for PUP-8044 as part of this PR. That is because these changes to upgrade openssl on rhel7 are pre-requisite for the changes in PUP-8044.

File: acceptance/setup/aio/pre-suite/010_Install.rb [hunk 1]
```diff
@@ -5,6 +5,22 @@
 
 test_name "Install Packages"
 
+def func_use_system_openssl?
+  # Presently .i.e as of Oct 22 there is no way to specify an environment variable
+  # in PA CI so we are going to hard code it till there is support for FIPS platforms
+  return true
```
Comment (MikaelSmith) [inline acceptance/setup/aio/pre-suite/010_Install.rb]: This doesn't make any sense to me... we don't want to use system openssl in all our tests. Also, it should be easy to add an environment variable in PA CI...

**Comment by MikaelSmith** [comment]
What are you aiming for here? Your changes can't be merged to master, because they remove a bunch of tests and only support testing with system openssl. As a feature branch it's fine.

**Comment by jsane** [comment]
I will be closing this PR. The way of detecting FIPS platforms and the custom actions based off it is anyway going to be different. Per discussions with the platform os team and others it was deemed that using actual platform names to denote FIPS was best long term approach. In addition, it was decided that the FIPS platforms would actually be FIPS enabled and hence will obviate the need for moving to FIPS mode from within scripts (PUP-8044).
Also we do want all acceptance tests enabled, after updating relevant ones to be FIPS compatible, when testing in FIPS mode. Their removal was temporary; that experimentation should not have been part of this PR.

**Pull request closed without merging**